### PR TITLE
Add storybook snapshot update command to testing documentation

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Description

This came up in https://github.com/WordPress/openverse/pull/3727#issuecomment-1919615370.

This PR adds the storybook snapshot update command to the frontend testing documentation, so folks updating aspects of the frontend have all the necessary commands readily available in the same document.

I also removed all the preceding `$` characters from these commands. Because they're of the `console` type (or perhaps just a code block in general), Sphinx makes it easy to copy the commands from the browser and paste into a shell. However, those commands come with the leading `$` character too which makes them hard to paste and run immediately. This PR removes them all for this document, I'll have a follow up PR that removes them everywhere else. 

## Testing Instructions

View the generated documentation output and verify it looks correct.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by obulat — APPROVED** [review]
Thank you for updating the documentation!

**Comment by AetherUnbound** [comment]
Okay, it looks like there's a difference in what's rendered when you remove the preceding `$` 😕 

**Before**
image

**After**
image


It copies easier, but it looks like all syntax is lost...going to try and see why that's the case. Otherwise I'll remove that commit from the PR.

File: documentation/frontend/guides/test.md [hunk 1]
```diff
@@ -17,39 +17,45 @@ Storybook visual regression tests, read
 
 2. Run unit tests for the frontend.
 
-   ```console
```
Comment (dhruvkb) [inline documentation/frontend/guides/test.md:20]: This is present in many other files too, would you mind opening an issue to fix these? It'll be a simple change that hopefully gives a new contributor a reason to set things up locally!

**Review by dhruvkb — APPROVED** [review]
LGTM! It'll be helpful to also have documentation on how to update the snapshots for a single test.

File: documentation/frontend/guides/test.md (hunk 1 above)
Comment (AetherUnbound) [reply documentation/frontend/guides/test.md:20]: I'm about to drop another PR that fixes these and adds a pre-commit check for it, actually!

**Pull request merged**